mlflow/deployments/interface.py

```diff
@@ -41,6 +44,15 @@ def get_deploy_client(target_uri):
         # Delete our deployment
         client.delete_deployment("spamDetector")
     """
+    if not target_uri:
+        target = get_deployments_target()
+        if not target:
```

Comment:
@prithvikannan I'm not sure that this works since `get_deployments_target()` currently *throws* if the target is not found, instead of returning none. Can we update and add a test for `get_deploy_client(None)`?

Reply:
ah yes, havent tried this out yet -- just threw something together. will update and test